dynamic_sharing_service/dynamic_scenario_builder.go

```diff
@@ -45,6 +37,13 @@ func (builder *DynamicScenarioBuilder) WithCallbackEndpoint(endpoint string) *Dy
 
 // Build constructs the DynamicScenario
 func (builder *DynamicScenarioBuilder) Build() (DynamicScenario, error) {
+	if builder.scenario.extensions == nil {
+		builder.scenario.extensions = make([]interface{}, 0)
+	}
+	if builder.scenario.policy == nil {
+		policy, _ := (&DynamicPolicyBuilder{}).Build()
```

Comment:
We don't want to be ignoring the error value from Build() do we? 

i think gometalinter used to warn about this, i'm guessing golangci-lint doesn't

Reply:
I wasn't sure, since this is for an empty Builder there should be no way it can fail to build. I can go either way with this though.

Reply:
There should be no way it can fail to build now, but always best to have a check in case the code changes later on! 👍